modules/artifact/src/main/java/org/mule/runtime/module/artifact/api/classloader/RegionClassLoader.java

```diff
@@ -424,8 +425,25 @@ public final Enumeration<URL> findResources(final String name) throws IOExceptio
     return new CompoundEnumeration<>(enumerations.toArray(new Enumeration[0]));
   }
 
+  /**
+   * @deprecated since 4.5, use {@link #disposeFromOwnerClassLoader()} instead.
```

Comment:
the idea would be for client code to jaut call `#dispose` on the app/domain classlaoder, right?

Reply:
Yes, in the `disposeFromOwnerClassLoader()` Java doc I mention that the method is intended to be used by the owner class loader while being disposed.

Reply:
i would mention that in this deprecation note instead of having that indirection through docs